lib/Auth/Process/WebAuthn.php

```diff
@@ -136,8 +154,24 @@ public function __construct($config, $reserved)
             $this->defaultEnabled = false;
         }
 
+        if (array_key_exists('force', $config)) {
+            $this->force = $config['force'];
+        } else {
+            $this->force = true;
+        }
+        if (array_key_exists('attrib_toggle', $config)) {
+            $this->toggleAttrib = $config['attrib_toggle'];
+        } else {
+            $this->toggleAttrib = 'toggle';
```

Comment:
This scrutinizer remark seems relevant. Can you change the variable declaration?

Reply:
description has been changed and it's ok now